src/coreclr/vm/prestub.cpp

```diff
@@ -1294,7 +1294,7 @@ namespace
         Instantiation targetMethodInst;
         if (targetType->HasInstantiation())
         {
-            declClassInst = declaration->GetMethodTable()->GetInstantiation();
+            declClassInst = declaration->GetMethodTable()->GetTypicalMethodTable()->GetInstantiation();
```

Comment:
I have been looking at the implementation of `MethodTable::GetTypicalMethodTable`. 

It uses `LookupTypeDef` that returns the MethodTable if the type was loaded, but it can return null if it was not loaded yet.

Do we guarantee somewhere that the typical method table is loaded before we can come here?

Reply:
Ok, this should guarantee that: https://github.com/dotnet/runtime/blob/fbf46726097e766bb7b8a3eb94315d55185e9cf3/src/coreclr/vm/methodtable.cpp#L4358-L4362